Cargo.toml

```diff
@@ -12,6 +12,11 @@ readme = "README.md"
 
 [dev-dependencies]
 rand = "0.3.15"
+serde_json = "1.0"
+
+[dependencies]
+serde = "1.0"
```

Comment:
The crate is added, but why is it not listed as `extern crate`? Is it because of the recent changes in the 2018 Edition? If so, I'm not familiar.

Reply:
If I recall correctly (almost a year ago), `serde_derive` did require `serde` to work, but you did not actually need to call `extern crate` for it.